core/src/main/java/org/apache/calcite/util/format/FormatElementEnum.java

```diff
@@ -306,12 +446,22 @@ static Work get() {
      * https://issues.apache.org/jira/browse/CALCITE-6252.  This may be
      * specific to Java 11. */
     final DateFormat mmmmFormat = new SimpleDateFormat(MONTH.javaFmt, Locale.US);
-    final DateFormat sFormat = new SimpleDateFormat(FF1.javaFmt, Locale.ROOT);
-    final DateFormat ssFormat = new SimpleDateFormat(FF2.javaFmt, Locale.ROOT);
     final DateFormat sssFormat = new SimpleDateFormat(FF3.javaFmt, Locale.ROOT);
-    final DateFormat ssssFormat = new SimpleDateFormat(FF4.javaFmt, Locale.ROOT);
-    final DateFormat sssssFormat = new SimpleDateFormat(FF5.javaFmt, Locale.ROOT);
-    final DateFormat ssssssFormat = new SimpleDateFormat(FF6.javaFmt, Locale.ROOT);
     final DateFormat yyFormat = new SimpleDateFormat(YY.javaFmt, Locale.ROOT);
+    final DateFormat yyyyFormat = new SimpleDateFormat(YYYY.javaFmt, Locale.ROOT);
+
+    /** Util to return the full or abbreviated weekday name from date and expected TextStyle. */
+    public String getDayFromDate(Date date, TextStyle style) {
```

Comment:
should this be private

Reply:
Good catch, can be kept private updated it!